engine/apps/schedules/models/custom_on_call_shift.py

```diff
@@ -274,6 +276,56 @@ def event_is_finished(self):
 
         return is_finished
 
+    def _daily_by_day_to_ical(self, time_zone, start, users_queue):
+        """Create ical weekly shifts to distribute user groups combining daily + by_day."""
```

Comment:
Probably it makes sense to provide an example here in the comment so it's more clear what the method does